oval_graph/oval_tree/evaluate.py

```diff
@@ -113,23 +111,18 @@ def oval_operator_xor(result):
                 noteval
     """
     out_result = None
-    if (result['true_cnt'] % 2) == 1\
-            and eq_zero_unknown_noteval_notappl(result):
+    eval_error_cnt = eq_zero(result, 'error_cnt')
```

Comment:
It's not easy to see how `eq_zero(result, 'error_cnt')` is better than `result.num_errors == 0` (that would work if result is a named tuple) or `result.no_errors()` (that one would require result to be a class)

Reply:
I create a results class and change bad naming in another PR. I create an Issue about it.  #192